content/manuals/desktop/release-notes.md

```diff
@@ -23,6 +23,73 @@ Docker Desktop versions older than 6 months from the latest release are not avai
 
 Take a look at the [Docker Public Roadmap](https://github.com/orgs/docker/projects/51/views/1?filterQuery=) to see what's coming next.
 
+## 4.34.0
+
+{{< release-date date="2024-10-02" >}}
+
+{{< desktop-install-v2 all=true beta_win_arm=true version="4.35.0" build_path="/TBD/" >}}
+
+### New
+
+- Support for [Docker Desktop on Red Hat Enterprise Linux](/manuals/desktop/install/linux/rhel.md) is now generally available.
+- Volume Backup and Share is now generally available and can be found in the **Volumes** tab.
+- Terminal support within Docker Desktop using system shells is now generally available.
+- Enhanced support for IPv6, allowing users to configure their Docker VM for IPv4-only, dual-stack, or IPv6-only operation. See [IPv6 documentation](https://docs.docker.com/go/desktop-ipv6-only).
+- containerd image store:
+  - Improved output of `docker image inspect` to account for multi-platform images.
+  - Enables users to import and export a subset of multi-platform images.
```

Comment:
Is this really available?

- https://github.com/docker/cli/pull/5331

Reply:
Yeah, I don't think these are there yet; these will be in v28.0 of the engine.

Reply:
Effectively only changes in Docker Engine were a new flag on the daemon (`--features`), but that cannot be used by Docker Desktop, and an updated version of BuildKit, but we don't know what's in it; https://docs.docker.com/engine/release-notes/27/#2730